### ISSUE TITLE
[Installation] - Unable to install following instructions.

### ISSUE THREAD

**Issue by shawn-crossley** [negativetwelve/react-native-lookback#18]
When following the instructions from the readme.  You get the following error

```
Fetching podspec for `react-native-lookback` from `../node_modules/react-native-lookback`

[!] Invalid `react-native-lookback.podspec` file: undefined method `dig' for #<Hash:0x007fb6e4ebf120>.
```

I downgraded to the version 1.0 and followed the instructions there and it worked

**Comment by negativetwelve** [comment]
thanks for reporting! what version of ruby are you running? I keep forgetting that `dig` is only available in 2.3...

I'll make a fix for that right now!

**Comment by negativetwelve** [comment]
I've deployed version 1.2.3 with the podspec fix, however, I'm actually running into issues linking the official LookbackSafe framework with this pod. There's a configuration issue with the Lookback official iOS sdk that I'm working with the team on.

For now, I've updated the readme with instructions on how to link the pod manually and released 1.2.3 to npm.

https://github.com/negativetwelve/react-native-lookback/releases/tag/1.2.3

Let me know if that's okay of if there's anything else I can clarify!
